bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -292,8 +333,8 @@ fn main() -> Result<()> {
                                     "Runtime done for request_id: {:?} with status: {:?}",
                                     request_id, status
                                 );
-                                logs_agent.flush();
-                                dogstats_client.flush();
+                                logs_flusher.flush().await;
+                                statsd_flusher.flush().await;
```

Comment:
Can `statsd_flusher.flush()` start before the `logs_flusher.flush()` is awaited, to do them concurrently?

Reply:
they are now concurrent w/ `tokio::join`